Dashboard/.eslintrc

```diff
@@ -0,0 +1,83 @@
+{
+  "extends": "airbnb",
```

Comment:
This is based on some styling from airbnb? I guess its relatively stable right? i.e. we wont have broken checks if airbnb decides to refurbish?

Reply:
yes this is one of the most commonly used styleguides - if not THE most used. no need to worry

Reply:
also the version is locked so it wont just suddenly break